backend/src/bcws/bcws.controller.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  ClassSerializerInterceptor,
+  Controller,
+  Get,
+  Inject,
+  UseInterceptors,
+} from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { BcwsService } from './bcws.service';
+import { Program, TokenType } from '../auth/interface';
+import { Programs } from '../auth/program.decorator';
+import { Token } from '../auth/token.decorator';
+
+import { AppLogger } from '../logger/logger.service';
+
+@ApiTags('BcwsController API')
+@UseInterceptors(ClassSerializerInterceptor)
+@Programs([Program.BCWS])
+@Controller('bcws')
+export class BcwsController {
```

Comment:
I don't love this having its own controller

Reply:
I thought we would need a controller for BCWS anyways since we will have BCWS specific endpoints for the dashboard and the profile as well. I agree on not having a controller for a single endpoint though.